scripts/test-docker.sh

```
@@ -0,0 +1,144 @@
+echo ""
+
+read -p "エンドツーエンドテストを実行しますか？ (y/N): " -n 1 -r
+echo ""
+
+if [[ $REPLY =~ ^[Yy]$ ]]; then
+  echo -e "${BLUE}エンドツーエンドテスト実行中...${NC}"
+
+  # 既存のデータとスクリーンショットをバックアップ
+  if [ -d data ]; then
+    echo "既存のデータをバックアップ中..."
+    cp -r data data.backup.$(date +%s)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)